pkgs/development/compilers/go/binary.nix

```diff
@@ -38,4 +38,14 @@ stdenv.mkDerivation rec {
     ln -s $out/share/go/bin/go $out/bin/go
     runHook postInstall
   '';
+
+  meta = with lib; {
```

Comment:
`lib`is not imported - this breaks at least `maintainers/scripts/update.nix` maybe more ⚠️